bootstrap/examples/kubeflow_toolbox.yaml

```diff
@@ -0,0 +1,66 @@
+# Deploy kubeflow to namespace "kubeflow"
+
+# TODO(inc0): Create rbac role bindings
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: kubeflow
+
+# Headless service because StatefulSet requires it
+---
+kind: Service
+apiVersion: v1
+metadata:
+  name: kubeflow-toolbox
+  namespace: kubeflow
+spec:
+  clusterIP: "None"
+
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: kubeflow-toolbox-pvc
+  namespace: kubeflow
+  labels:
+    app: kubeflow-toolbox
+spec:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 5Gi
+
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: kubeflow-toolbox
+  namespace: kubeflow
+spec:
+  selector:
+    matchLabels:
+      app: kubeflow-toolbox
+  serviceName: kubeflow-toolbox
+  template:
+    metadata:
+      name: kubeflow-toolbox
+      labels:
+        app: kubeflow-toolbox
+    spec:
+      containers:
+      - name: kubeflow-toolbox
+        image: gcr.io/kubeflow-images-public/bootstrapper
+        env:
+        - name: NAMESPACE
+          value: "kubeflow"
+        - name: DEPLOY_JOB
+          value: "1"
```

Comment:
Why 1? Why not true/false?

Reply:
it has to be string with whatever in it. Can be true